nemo/collections/asr/models/transformer_bpe_models.py

```diff
@@ -206,6 +227,48 @@ def transcribe(
         if paths2audio_files is None or len(paths2audio_files) == 0:
             return {}
 
+        if return_hypotheses:
+            logging.warning("return_hypotheses=True is currently not supported, returning text instead.")
+
+        if isinstance(paths2audio_files, list):
+            logging.info(f"Found paths2audio_files to be a list of {len(paths2audio_files)} items.")
+            logging.info(f"Assuming each item in paths2audio_files is a path to audio file.")
+            logging.info(f"Transcribing with default Canary setting of English without PnC.")
```

Comment:
I think this will print each time for a new batch? Can we suppress this and only print once

Reply:
In HF-Leaderboard, we call "transcribe" repeatedly for each batch. However, I believe we can move this to debug instead of INFO. Otherwise, these would print for each transcribe, and the user experience wouldn't be nice.